squad-create-reproducer

```diff
@@ -54,6 +54,12 @@ def parse_args(raw_args):
         nargs="+",
         help="The list of accepted build names (for example, gcc-12-lkftconfig). Regex is supported.",
     )
+    parser.add_argument(
+        "--suite-name",
+        required=False,
+        default="ltp-syscalls",
```

Comment:
Maybe we should always ask the user to submit a suite-name, and say 'required=True,' ?